specs/protocol/isthmus/exec-engine.md

```diff
@@ -131,6 +132,13 @@ is applicable for scenarios where the actual withdrawals root value is not readi
 
 Withdrawals list in the block body is encoded as an empty RLP list.
 
+## Block Sealing
+
+[EIP-7251](https://eips.ethereum.org/EIPS/eip-7251) introduces a new consolidation
+[EIP-7685](https://eips.ethereum.org/EIPS/eip-7685) request with type `CONSOLIDATION_REQUEST_TYPE` (or `0x02`).
```

Comment:
does it make sense to use the word 'upgrade of' instead of 'consolidation' here?

Reply:
Less familiar with L1 jargon so happy to defer to your opinion here. I tried to keep this as close to the eip as possible

Reply:
Where is the request of type 0x02 defined? It's not in the eip-7685 spec.
And do we want to specify any behavior for other (possibly unknown) request types?